### ISSUE TITLE
Use bunyan

### ISSUE THREAD

**Issue by njb-said** [CraftTogether/CraftWeb#3]
It could be useful to use bunyan at some point, @oliverdunk can explain more about it 😛

**Comment by oliverdunk** [comment]
Definitely, really great for production login and you can use request IDs in order to get information on a particular error report from a user.

**Comment by theminecoder** [comment]
Can you provide a link to this?

**Comment by jakeycodes** [comment]
i think this it https://github.com/trentm/node-bunyan

**Comment by destruc7i0n** [comment]
This seems like a cool thing... I may use it for my own projects.

**Comment by oliverdunk** [comment]
@destruc7i0n I actually did a blog post which explain a potential use case: https://www.mcjukebox.net/2016/03/19/bunyan.html
